.github/workflows/remove.yml

```diff
@@ -0,0 +1,115 @@
+name: Template Removal
+
```

Comment:
Just for consistency let's have file name `.github/workflows/remove-template.yml`